### DISCUSSION TITLE
Getting Error: Missing CAP cache value for: NativeCodeInfo:RISCV64LibCHelperDirectives:StructInfo:CPUFeatures

### DISCUSSION THREAD

**Discussion by saradhipb** [oracle/graal#9455] [Q&A]
I am very new to GraalVM tool, While run simple HelloWorld.java getting this error. Am using java 17.0.11 2024-04-16 LTS
Java(TM) SE Runtime Environment Oracle GraalVM 17.0.11+7.1 (build 17.0.11+7-LTS-jvmci-23.0-b34)
Java HotSpot(TM) 64-Bit Server VM Oracle GraalVM 17.0.11+7.1 (build 17.0.11+7-LTS-jvmci-23.0-b34, mixed mode, sharing)

GraalVM Native Image: Generating 'helloworld' (executable)...
========================================================================================================================
For detailed information and explanations on the build output, visit:
https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/BuildOutput.md
------------------------------------------------------------------------------------------------------------------------

[1/8] Initializing...                                                                                    (0.0s @ 0.09GB)
Error: Missing CAP cache value for: NativeCodeInfo:RISCV64LibCHelperDirectives:StructInfo:CPUFeatures
Error: Use -H:+ReportExceptionStackTraces to print stacktrace of underlying exception
------------------------------------------------------------------------------------------------------------------------
                        0.2s (4.8% of total time) in 10 GCs | Peak RSS: 0.46GB | CPU load: 2.24
========================================================================================================================
Finished generating 'helloworld' in 4.4s.

How to get out from this error?

**Comment by alina-yur** [comment]
hi @saradhipb, I'll convert your question to an issue, if you can provide more context about your build environment etc there, that would be great.

**Reply by saradhipb** [reply]
I am using VSCode Community Edition 2022.
Am using java 17.0.11 2024-04-16 LTS
Java(TM) SE Runtime Environment Oracle GraalVM 17.0.11+7.1 (build 17.0.11+7-LTS-jvmci-23.0-b34)
Java HotSpot(TM) 64-Bit Server VM Oracle GraalVM 17.0.11+7.1 (build 17.0.11+7-LTS-jvmci-23.0-b34,
Windows 10 Enterprise
Version 22H2
OS Build 1905.4651

Let me know if you need additional info

**Reply by Zeavee** [reply]
Hi, I'm sorry, but I cannot reproduce the issue.

I suppose you downloaded the JDK from https://www.oracle.com/java/technologies/javase/graalvm-jdk17-archive-downloads.html ?

If you did, can I please know what commands did you use?
